content/en/developers/integrations/log_integration.md

```diff
@@ -91,17 +101,29 @@ To add a facet or measure:
 
 **Note:** You do not need to create facets for standard attributes. Facets for standard attributes will be added automatically by Datadog when the log pipeline is published. See the [default standard attribute list][6] for the standard Datadog attributes under their specific groups.
 
-To easily navigate the facet list, group similar facets together. For fields specific to the integration logs, create a group with the same name as the `source` tag. 
+To easily navigate the facet list, facets are grouped together. For fields specific to the integration logs, create a single group with the same name as the `source` tag. 
 
 1. In the log panel, click the Cog icon next to the attribute that you want in the new group.
 2. Select **Edit facet/measure for @attribute**. If there isn't a facet for the attribute yet, select **Create facet/measure for @attribute**.
 3. Click **Advanced options**.
-4. In the **Group** field, enter the name of the new group, and select **New group**.
+4. In the **Group** field, enter the name and a description of the new group, and select **New group**.
 5. Click **Update**.
 
+**Requirements:**
+- Use standard attributes as much as possible. Otherwise, use a specific namespace (such as the integration name) to avoid conflict as facets are shared across Datadog.
```

Comment:
```suggestion
- Use standard attributes as much as possible. Otherwise, use a specific namespace (such as the integration name) to avoid conflict, because facets are shared across Datadog.
```